README.md

```diff
@@ -1,2 +1,69 @@
-# SLMM
-The coding exercise for server side applicants
+SLMM - The coding exercise for server side applicants
+=====================================================
+
+Step 1 – all server-side positions
+----------------------------------
+Assume we are interested in creating a smart lawn mowing machine called SLMM (smart lawn mowing machine). Your task is to create the software that will run in the SLMM itself and will be responsible of doing the following actions:
+
+1. Turn the SLMM 90o left or right -> This should take 15 seconds to do
+2. Move forward by one position -> This takes 30 seconds to do
+3. Mow the lawn at the current position -> This takes 120 seconds to do
+
+The application you will create should be able to accept the above commands and do them. Since you do not have access to the actual hardware, for now the only thing required is to accept input from console and output the result (the action being done along with the position of the SLMM) in the console. During application startup, the SLMM is given a set of dimensions (passed in as command line parameters) that represent the width and length of the garden that the SLMM is in.
+
+### Commands accepted
+1. TL -> Turn Left
+2. TR -> Turn Right
+3. MF -> Move Forward
+4. ML -> Mow Lawn
+
+### Deliverable
+One console application that represents the application as described. You should provide access to an online repository, based on this one that contains all source code for this application, along with complete instructions on how to build your application locally. If any tests or libraries were written, please provide code for all and full instructions on how to run the build and tests.
+
+### Acceptance criteria
+1. The SLMM never goes outside of the dimensions of the garden as supplied during startup
+2. The output of the SLMM after it finished each action must be in the format: “{Time} – {Action taken place} – {current location of SLMM}”
```

Comment:
The output for this step would not show the end time of an action as it as been specified in the step2 acceptance criteria?